pkg/feeder/feeder.go

```diff
@@ -374,15 +379,15 @@ func (c Client) GetFullContractRaw(contractAddress, blockHash, blockNumber strin
 	if err != nil {
 		metr.IncreaseFullContractsFailed()
 		metr.IncreaseRequestsFailed()
-		Logger.With("Error", err, "Gateway URL", c.BaseURL).Error("Unable to create a request for get_full_contract.")
+		c.logger.Error("Unable to create a request for get_full_contract.", "error", err, "url", c.BaseURL)
```

Comment:
```suggestion
		c.logger.Error("Unable to create a request for get_full_contract.", "error", err, "url", c.BaseURL)
```

Reply:
It appears no change is suggested--I tried to remove the rest of the periods from the log messages in a recent commit.